### ISSUE TITLE
Touch gesture timeouts on Xcode 6.1 Simulators

### ISSUE THREAD

**Issue by JaniJegoroff** [calabash/calabash-ios#566]
**Environment:**
- Calabash 0.11.1
- run_loop 1.0.9
- Xcode 6.1
- 'iPad Retina (8.1 Simulator)'
- Yosemite 10.10

**Steps to reproduce:**
1. Build app to `iPad Retina (8.1 Simulator)`
2. Launch `calabash-ios console`
3. Execute: `start_test_server_in_background`
4. Execute: `touch("view marked:'<your-text>'"`

**Console trace:**

```
Janis-Air:myApp janijegoroff$ DEBUG=1 DEBUG_HTTP=1 DEVICE_TARGET='iPad Retina (8.1 Simulator)' calabash-ios console
Running irb...
irb(main):001:0> start_test_server_in_background

INFO: Using uia strategy: 'preferences'
Sending 'QUIT' to instruments process '3169'
Waiting for instruments '3169' to terminate
Preparation took 2.950325 seconds
{
                        :app => "/Users/janijegoroff/Library/Developer/Xcode/DerivedData/myApp-civuqgdjemtasdaqyhpsrgakmnyv/Build/Products/Calabash-iphonesimulator/myApp.app",
                       :args => [],
    :bundle_dir_or_bundle_id => "/Users/janijegoroff/Library/Developer/Xcode/DerivedData/myApp-civuqgdjemtasdaqyhpsrgakmnyv/Build/Products/Calabash-iphonesimulator/myApp.app",
                  :bundle_id => "myBundleId",
                     :device => "iphone",
              :device_target => "iPad Retina (8.1 Simulator)",
              :launch_method => :instruments,
             :launch_retries => 5,
                   :log_file => "/var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0/run_loop.out",
                  :no_launch => false,
                    :no_stop => false,
                      :reset => false,
                :results_dir => "/var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0",
          :results_dir_trace => "/var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0/trace",
                     :script => "/var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0/_run_loop.js",
                :sdk_version => nil,
                       :udid => "iPad Retina (8.1 Simulator)",
               :uia_strategy => :preferences,
                      :xcode => "6.1",
                 :xcode_path => "/Applications/Xcode61/Xcode.app/Contents/Developer"
}

### Starting on iPad Retina (8.1 Simulator) App: /Users/janijegoroff/Library/Developer/Xcode/DerivedData/myApp-civuqgdjemtasdaqyhpsrgakmnyv/Build/Products/Calabash-iphonesimulator/myApp.app ###
2014-10-20 23:02:19 +0100 xcrun instruments -w "iPad Retina (8.1 Simulator)" -D "/var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0/trace" -t "Automation" "/Users/janijegoroff/Library/Developer/Xcode/DerivedData/myApp-civuqgdjemtasdaqyhpsrgakmnyv/Build/Products/Calabash-iphonesimulator/myApp.app" -e UIARESULTSPATH /var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0 -e UIASCRIPT /var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0/_run_loop.js  &> /var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0/run_loop.out
Launching took 8.111403 seconds
Calabash::Cucumber::Launcher: Launch Method instruments
Log file: /var/folders/66/62hrz8650r72_srksbdfwpjm0000gn/T/run_loop20141020-3233-p170g0/run_loop.out
irb(main):002:0> touch("view marked:'Main menu'")
= Request

! CONNECT TO localhost:37265
! CONNECTION ESTABLISHED
POST /map HTTP/1.1
Content-Type: application/x-www-form-urlencoded
User-Agent: HTTPClient/1.0 (2.3.4.1, ruby 2.1.2 (2014-05-08))
Accept: */*
Date: Mon, 20 Oct 2014 22:02:42 GMT
Content-Length: 86
Host: localhost:37265

{"query":"view marked:'Main menu'","operation":{"method_name":"query","arguments":[]}}

= Response

HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Content-Length: 460
Access-Control-Allow-Credentials: true
Access-Control-Allow-Methods: GET, POST, PUT, DELETE, OPTIONS
Date: Mon, 20 Oct 2014 22:02:42 GMT
Accept-Ranges: bytes
Access-Control-Max-Age: 3000

{"status_bar_orientation":"down","results":[{"text":"Main menu","enabled":true,"rect":{"center_x":392,"y":310,"width":516,"x":134,"center_y":323.75,"height":27.5},"id":null,"description":"<UITableViewLabel: 0x7b7f9af0; frame = (20 8; 516 27.5); text = 'Main menu'; userInteractionEnabled = NO; layer = <_UILabelLayer: 0x7b7f9bb0>>","label":"Main menu","alpha":1,"class":"UITableViewLabel","frame":{"y":8,"width":516,"x":20,"height":27.5}}],"outcome":"SUCCESS"}Sending UIA command
uia.tapOffset('{:x 392, :y 323.75}')
= Request

POST /uia HTTP/1.1
Content-Type: application/x-www-form-urlencoded
User-Agent: HTTPClient/1.0 (2.3.4.1, ruby 2.1.2 (2014-05-08))
Accept: */*
Date: Mon, 20 Oct 2014 22:02:42 GMT
Content-Length: 50
Host: localhost:37265

{"command":"uia.tapOffset('{:x 392, :y 323.75}')"}

= Response

HTTP/1.1 200 OK
Transfer-Encoding: chunked
Accept-Ranges: bytes
Date: Mon, 20 Oct 2014 22:02:42 GMT

6c
{"details":"","outcome":"FAILURE","reason":"Timed out running command uia.tapOffset('{:x 392, :y 323.75}')"}
0

RuntimeError: uia action failed because: Timed out running command uia.tapOffset('{:x 392, :y 323.75}')

    from /Users/janijegoroff/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.11.1/lib/calabash-cucumber/uia.rb:31:in `uia'
    from /Users/janijegoroff/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.11.1/lib/calabash-cucumber/uia.rb:319:in `uia_handle_command'
    from /Users/janijegoroff/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.11.1/lib/calabash-cucumber/uia.rb:138:in `uia_tap_offset'
    from /Users/janijegoroff/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.11.1/lib/calabash-cucumber/actions/instruments_actions.rb:93:in `query_action'
    from /Users/janijegoroff/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.11.1/lib/calabash-cucumber/actions/instruments_actions.rb:15:in `touch'
    from /Users/janijegoroff/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.11.1/lib/calabash-cucumber/core.rb:1095:in `query_action_with_options'
    from /Users/janijegoroff/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.11.1/lib/calabash-cucumber/core.rb:174:in `touch'
    from (irb):2
    from /Users/janijegoroff/.rbenv/versions/2.1.2/bin/irb:11:in `<main>'
irb(main):003:0>
```

**Comment by lindanordstrom** [comment]
Hi, sorry for commenting on such an old issue, but I am experiencing problems with this. I have a few tests where I have set RESET_BETWEEN_SCENARIOS=1, so the user is always signed out when a test starts. But in some scenarios we want to relaunch the app as part of a test, where we don't want to app to be reset, so I am calling the relaunch function (with :uia_strategy => :host, :reset => false). 

The thing is I can run the scenario by itself without problem, but if I run a similar/identical scenario as the second scenario in a feature, cucumber will timeout on the "Trying write of command 13:uia.tapOffset('{:x 28, :y 41}') at index 13"

I've compared the logs to see what actually differs when I run the identical test twice. From what I can see the one that works fine will output:
"Trying write of command 1:uia.tapOffset"
And the one that fails outputs:
Trying write of command 13:uia.tapOffset

Adding a screenshot where I compare the working scenario with the failing scenario
And adding the full log

The tests are identical and does the following:
* open app
* sign in
* relaunch app
* press menu icon

![screen shot 2016-02-11 at 14 44 57](https://cloud.githubusercontent.com/assets/13778933/12979676/a2f3e14a-d0d0-11e5-9d69-219c0f78a191.png)
[errorLog.txt](https://files.zenhub.io/56bca2da95307a0978e7ab3b)
Uploaded using [**ZenHub.io**](http://www.zenhub.io)
